### PR TITLE
#159082299 Upgrade Authors Haven Django app from Django 1.1.0 to Django 2.0.7

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
Upgrade Authors Haven Django app from Django 1.1.0 to Django 2.0.7
#### Description of Task to be completed
* Update the project dependencies
* Use django2x conventions in the code e.g change urls to path
#### Why is this important?
Django 2.x is the latest release of Django with full support of Python 3.x series.
#### Relevant Pivotal Tracker Stories
#159082299

### CODE REVIEW

File: authors/apps/authentication/backends.py [hunk 1]
```diff
@@ -8,3 +8,6 @@
 
 """Configure JWT Here"""
 
+class JWTAuthentication:
```
Comment (pluwum) [inline authors/apps/authentication/backends.py:11]: Are these changes part of the upgrade?

**Comment by pluwum** [comment]
@AGMETEOR Please have your PR title and the commit message follow the convention shared earlier.

File: authors/apps/authentication/backends.py (hunk 1 above)
Comment (RutaleIvanPaul) [reply authors/apps/authentication/backends.py:11]: Yes, they are. It's a way to deal with some errors we were getting.

Comment (AGMETEOR) [reply authors/apps/authentication/backends.py:11]: We needed to write JWTAuthentication class to remove errors that were preventing project from running.

**Pull request merged**